package.json

```diff
@@ -18,17 +18,17 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@types/node": "14.14.10",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "4.28.5",
-    "@typescript-eslint/parser": "4.28.5",
-    "eslint": "7.32.0",
-    "eslint-config-prettier": "8.3.0",
-    "eslint-plugin-prettier": "3.4.0",
-    "ethers": "5.7.2",
-    "prettier": "2.2.1",
+    "@types/node": "18.15.11",
+    "@types/uuid": "9.0.1",
+    "@typescript-eslint/eslint-plugin": "5.57.1",
+    "@typescript-eslint/parser": "5.57.1",
+    "eslint": "8.37.0",
+    "eslint-config-prettier": "8.8.0",
+    "eslint-plugin-prettier": "4.2.1",
+    "ethers": "6.2.3",
+    "prettier": "2.8.7",
     "ts-node": "10.9.1",
-    "typescript": "4.1.2",
+    "typescript": "5.0.3",
     "uuid": "9.0.0"
   },
   "peerDependencies": {
```

Comment:
shouldn't the peerDependency on `ethers` also be upgraded to 6.x?

Reply:
@raster21 Good callout, I've updated the peer dependency to match 👍